app/swagger/schemas/service_history.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Swagger
+  module Schemas
+    class ServiceHistory
+      include Swagger::Blocks
+
+      swagger_schema :ServiceHistory do
+        key :required, [:data]
+
+        property :data, type: :object do
+          key :required, [:attributes]
+          property :attributes, type: :object do
+            key :required, [:service_history]
+            property :service_history do
+              key :type, :array
+              items do
+                key :required, %i[branch_of_service begin_date]
+                property :branch_of_service, type: :string, example: 'Air Force'
+                property :begin_date, type: :string, example: '2007-04-01'
```

Comment:
Can we use a `date` type for this field?

Reply:
@jkassemi - For our other date or datetime API response values, the precedence that I have noticed has been to return a string (i.e. `"2018-02-27T14:41:32.283Z"`).

I confirmed with the frontend devs that returning the date as `'2007-04-01'` was desirable.  

Will that work?

Reply:
totally - there's just a `format` option you can pass to further specify the field formatting:

`property :begin_date, type: :string, format: :date, example: '2007-04-01'`

Reply:
ah, my bad.  will update...thanks!